rest_framework_extensions/serializers.py

```diff
@@ -28,7 +28,10 @@ def save(self, **kwargs):
 
     def update(self, instance, validated_attrs):
         for attr, value in validated_attrs.items():
-            setattr(instance, attr, value)
+            if hasattr(getattr(instance, attr), 'set'):
```

Comment:
```python
if attr != 'update_fields' and hasattr(getattr(instance, attr), 'set'):
```
or
```python
if hasattr(getattr(instance, attr, None), 'set'):
```
should fix https://travis-ci.org/chibisov/drf-extensions/jobs/350934079#L481

Reply:
yes only 1.11+ supported